src/test/ui/consts/const_forget.rs

```diff
@@ -0,0 +1,22 @@
+// run-pass
```

Comment:
Since this is a const-test, why is `check-pass` not sufficient?

Reply:
@DutchGhost this blocks enabling leak checks again in https://github.com/RalfJung/miri-test-libstd, would be good if we could land this quickly. :)